Source/WebCore/platform/graphics/FontCascadeFonts.cpp

```diff
@@ -502,22 +502,29 @@ GlyphData FontCascadeFonts::glyphDataForCharacter(UChar32 c, const FontCascadeDe
     ASSERT(m_thread ? m_thread->ptr() == &Thread::current() : isMainThread());
     ASSERT(variant != AutoVariant);
 
+    FontCascadeDescription fontDescription(description);
```

Comment:
Why do we clone the `FontCascadeDescription` here?

Reply:
I suspect not cloning (e.g. editing the original description directly) and removing the special cases for `from-font` at https://searchfox.org/wubkat/rev/42dc4893aca2f5a0e36c4314c8aa0555ebce6c88/Source/WebCore/css/ComputedStyleExtractor.cpp#366-369 may solve the computed style

Reply:
> I suspect not cloning (e.g. editing the original description directly).

FontCascadeFonts::glyphDataForCharacter and its caller chain (e.g., FontCascade::glyphDataForCharacter) are const methods. We need to remove the tailing const from them. Do you mean it? Please let me know if there is another way to directly modify the description. 

> removing the special cases for from-font at https://searchfox.org/wubkat/rev/42dc4893aca2f5a0e36c4314c8aa0555ebce6c88/Source/WebCore/css/ComputedStyleExtractor.cpp#366-369 may solve the computed style

Yes, I have a fix for that, but prefer solving one issue per ticket.